### PR TITLE
Fe#14 data selection

### PR DESCRIPTION
**Pull request by stripes416**

### CODE REVIEW

**Comment by something15525** [comment]
@stripes416 is this supposed to just show the data values and get rid of the map when items are selected?

File: js/dashboard.js [hunk 1]
```diff
@@ -199,6 +199,110 @@ $(function() {
...
 
+    $('#leafwetness-button').click(function() {
+				$.ajax({
+            url: 'http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness',
+            type: "GET"
+        }).done(function(json) {
+            if (json.env_data) {
+                window.location.href = "http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness";
+            } else {
+                // go nowhere
```
Comment (something15525) [inline js/dashboard.js]: These "go nowhere" statements seem unnecessary. shouldn't we just do nothing at all? or is there a reason we're specifying a window location to no where?

File: js/.idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (something15525) [inline js/.idea/vcs.xml]: Personal opinion, but the .idea folder should be completely ignored.

File: js/dashboard.js (hunk 1 above)
Comment (stripes416) [inline js/dashboard.js]: I didn't know what to do. Is a pop up error message appropriate here? Should we do nothing at all?

File: js/.idea/vcs.xml (hunk 2 above)
Comment (stripes416) [inline js/.idea/vcs.xml]: Sure

**Comment by stripes416** [comment]
Again, I didn't know what to do. The next step is for someone to take the data that's retrieved and actually generate the map with it.

File: js/dashboard.js (hunk 1 above)
Comment (stripes416) [inline js/dashboard.js]: Okay, after coffee, I'm thinking more clearly. I added those as basic placeholders. Didn't we say in the design phase that we would error handle? Depending on when the error occurs, shouldn't we have it go to a basic "here's the thing that went wrong, sorry. Try to reload the page in a few minutes" page?

Comment (something15525) [inline js/dashboard.js]: But we're not including that for MVP, right? I don't think placeholders are necessary right now. And if we do leave them in there, we should add a TODO line.

File: js/.idea/vcs.xml (hunk 2 above)
Comment (something15525) [inline js/.idea/vcs.xml]: Okay, I'll add a commit removing them.

**Comment by something15525** [comment]
The only change I would still make is to add TODOs to the areas that "go nowhere", then I think it should be good to merge!

**Comment by scotte216** [comment]
It works for me: Clicking the various icons will display the appropriate data in text form.

 I think having error handling like this is important to include, even in the MVP. If it's not able to connect to the server, it's nice to have a little feedback as to why things aren't working. It doesn't have to be anything fancy. Just an 'oops, can't reach the server" or 'something went wrong' would suffice. 

EDIT: But creating those error message/pages could be a different ticket.

**Comment by something15525** [comment]
@scotte216 I think the error handling will be in the other ticket, where we add data visualization, or it should be it's own ticket altogether. I'm going to merge this if there are no objections.

**Pull request merged**